.github/workflows/ci.yml

```diff
@@ -5,10 +5,22 @@ on:
     branches:
       - dev
       - master
+    paths-ignore:
+      - 'README*.md'
+      - 'fastlane/**'
+      - 'assets/**'
+      - .github/PULL_REQUEST_TEMPLATE.md
```

Comment:
why do the other lines have single ticks '' but this one doesn't?

Reply:
The other ones have stars in them, which are reserved characters in YAML.